fastlane/lanes/localization-lanes.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+#################################################
+# Constants
+#################################################
+
```

Comment:
Right. This is because I initially thought I'd move more things to that file (especially lanes) already as part of this PR before deciding I'd do that reorg/move in a separate PR later. So the "Constants" header was differentiate that section from the other "Lanes" section… which ended up not getting moved here just yet.

I'll rearrange all that (and consider dropping the section title if it doesn't help) when I'll move more stuff to that file in my next PR.

Note that I generally agree that large verbose headers add more noise than anything especially if there are too many of them. Which is why I was also thinking of making all the doc comments headers that we have above every single lane in our Fastfiles be shorter, and looking more like YARD doc (or maybe focusing on more useful `desc` blocks) and without the lengthy `#####…` runs in them, to make them more readable, more doc-like and less noisy. And thus only keep 2 or 3 `#####…`-like separators **at most** in whole file to separate big sections like constants vs lanes (using them much like `// MARK: - Section divisor` would be used in Xcode… except if you know of a similar magic marker recognized by RubyMine and/or VSCode we could use for those instead)

Anyway, TL;DR this will be revisited in next PR anyway 🙂